XmlParser.lua

```diff
@@ -363,19 +363,23 @@ end
 
 local function parseTagType(self, xml, f)
     -- Test for tag type
+    local tag
```

Comment:
In fact, we don't need this variable because not all methods return a tag (such as the parseCdata). You can also check that where the parseTagType is called, the return is not used.